src/Pelagos/Bundle/AppBundle/Resources/public/js/data-discovery.js

```diff
@@ -226,8 +226,7 @@ function createRow(data, row)
         $(rowContent).find("#dataset-year").text(data["year"]);
     }
 
-    $(rowContent).find("#dataset-title").attr("href", $(rowContent).find("#dataset-title").attr("href").replace("placeholder-udi", data["udi"]) );
-    $(rowContent).find("#dataset-title").text(data["title"]);
+    $(rowContent).find("#dataset-title").replaceWith(data["title"]);
```

Comment:
Removing the href attr is fine, but replacing the the content of the element will remove the span and the ID. All that will be left is just the text. Which removes that SPAN, and ID, thus invalidating the element for later use.
Just keep .text, get rid of the .replaceWith.

Reply:
.text() worked as expected. I was hung up with a local dev environment issues.